### ISSUE TITLE
Pause current media

### ISSUE THREAD

**Issue by not-matthias** [mrexodia/WorkraveQt#37]
You can get a break while you are listening to music or watching a video. It would be nice if the media is paused so that one can continue watching/listening after the break.

**Should the media automatically be resumed?** I'm kinda against it, because you might leave your computer during a longer break.

**Comment by mrexodia** [comment]
Yeah makes sense, on Windows you can do this from the lock screen
